pkg/kubelet/eviction/interfaces.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Question: Is `interfaces.go` the name that we plan on using henceforth instead of `types.go`?